src/pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -105,6 +105,9 @@ def _set_ckpt_path(
         ft_checkpoints = [cb for cb in self.trainer.callbacks if isinstance(cb, _FaultToleranceCheckpoint)]
```

Comment:
You split this up, I would put this together with the `if` that uses it

Reply:
I'm not sure I follow? It's being used in multiple `if`s, first usage is on line 111

Reply:
https://github.com/Lightning-AI/lightning/pull/14911/commits/89a1fc8a52876423b56970e87912a53d9b4a3ea5